Automattic/wp-calypso
# Hosting Configuration: Add 'Enable SSH' behind a feature flag

Fixes #66577

## Proposed Changes

For sites with a plan that includes SSH, this pull request updates the `<SFTPCard>` to mention SSH and allow it to be enabled:

<img width="753" alt="image" src="https://user-images.githubusercontent.com/36432/184760463-13aa876d-f2db-41e7-9d4f-29035aaba2ce.png">

<img width="774" alt="image" src="https://user-images.githubusercontent.com/36432/184760329-1aad0e8f-6892-4eb9-ab5a-b7c4c4996d35.png">

Sites without SSH use the existing UI.

## Testing Instructions

1. Sandbox `public-api.wordpress.com`
2. Apply D85944-code and merge trunk to make sure the latest changes are applied.
3. Create a new Business plan site.
4. Navigate to `/hosting-config/<business-site>` and observe the updated UI.
5. Enable SSH and verify SSH access works as expected.
6. Disable SSH and verify SSH access is disabled.
7. Create a new Pro plan site.
8. Navigate to `/hosting-config/<business-site>` and observe UI limited to SFTP access.
